### ISSUE TITLE
Plone 4 and 5: navigation and sitemap does not meet accessibility compliant to screen reader users

### ISSUE THREAD

**Issue by ivanteoh** [plone/Products.CMFPlone#1535]
Recently one of our 4.x plone sites was audited for WCAG 2.0 AA. The report came back and mentioned that navigation and sitemap need to have hidden structural labels. Below are explanation on 'hidden structural labels' from the agent.

Hidden structural labels are specific text for screen reader users only, which describe the content. Nested lists are hierarchical, but their hierarchy is not clear to screen reader users. Therefore you need to use a hidden structural label to indicate the beginning and end of sub-navigation. For code examples see: http://maxdesign.com.au/articles/about-structural-labels/

For examples where hidden structural labels have been employed in navigation see:
http://www.adcq.qld.gov.au/about-us/the-commissioner (left-hand navigation)
http://www.melbourne.vic.gov.au/business/doing-business/Pages/doing-business-in-melbourne.aspx

Examples of hidden structural labels in a sitemap:
http://www.adcq.qld.gov.au/site-map

**Comment by jensens** [comment]
Closing as outdated (10 years old). Plone 4/5 navigation accessibility concerns. Plone 6 has entirely new Classic UI and Volto frontends with different navigation rendering.
If you think this issue is still relevant please reopen it. Thank you for the original report!
